migration_pipeline.py

```diff
@@ -38,22 +35,72 @@ def build_pipeline() -> Pipeline:
         os.makedirs(client_config['export_dir'], exist_ok=True)
 
     checkpoint_service = CheckpointService(client_config)
+    if args.export_pipeline:
+        return build_export_pipeline(client_config, checkpoint_service, args)
+
+    if args.import_pipeline:
+        return build_import_pipeline(client_config, checkpoint_service, args)
+
+    # Verification job
+    # TODO: Add verification job at the end
+
+
+def build_export_pipeline(client_config, checkpoint_service, args) -> Pipeline:
+    # All export jobs
```

Comment:
@mrchristine could you confirm if we missed any objects that should be exported/imported via the script?